Packs/MicrosoftExchangeOnline/ReleaseNotes/1_2_7.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS Extension Online Powershell v3
+
+- Fixed an issue where the ***ews-message-trace-get*** command failure when run it without the *sender_address* and *recipient_address* arguments.
```

Comment:
```suggestion
- Fixed an issue where the ***ews-message-trace-get*** command would fail when ran it without the *sender_address* and *recipient_address* arguments.
```